tests/test_replace_condition_ends_with.py

```diff
@@ -0,0 +1,169 @@
+            - Condition: EndsWith
+              KeyPath: UpdatedField
+              Value: banana
+        DisplayName: Test Title
+        Enabled: true
+        LogTypes:
+        - Windows.EventLogs
+        RuleID: {rule_id}
+        Tags: []
+        """
+        transformation = ReplaceConditionEndsWith(source_field_name="Field", target_field_name="UpdatedField")
+        rule = SigmaRule.from_yaml(raw_rule)
+        transformation.apply(pipeline, rule)
+
+        res = sigma_sdyaml_backend.convert(SigmaCollection(rules=[rule]))
+        assert yaml.safe_load(res) == yaml.safe_load(expected)
+
+    def test_with_multiple_values(self, pipeline, sigma_sdyaml_backend):
+        rule_id = uuid.uuid4()
+        raw_rule = f"""
+        title: Test Title
+        id: {rule_id}
+        logsource:
+            category: process_creation
+            product: windows
+        detection:
+            sel:
+                Field:
+                  - "banana"
+                  - "apple"
+            condition: sel
+        """
+
+        expected = f"""
+        AnalysisType: rule
+        Description: null
+        Detection:
+        - All:
+          - Condition: Equals
+            KeyPath: event_platform
+            Value: Windows
+          - All:
+            - Condition: Equals
+              KeyPath: event_simpleName
+              Value: ProcessRollup2
+            - Any:
+                - Condition: EndsWith
+                  KeyPath: UpdatedField
+                  Value: banana
+                - Condition: EndsWith
+                  KeyPath: UpdatedField
+                  Value: apple
+        DisplayName: Test Title
+        Enabled: true
+        LogTypes:
+        - Windows.EventLogs
+        RuleID: {rule_id}
+        Tags: []
+        """
+        transformation = ReplaceConditionEndsWith(source_field_name="Field", target_field_name="UpdatedField")
+        rule = SigmaRule.from_yaml(raw_rule)
+        transformation.apply(pipeline, rule)
+
+        res = sigma_sdyaml_backend.convert(SigmaCollection(rules=[rule]))
+        assert yaml.safe_load(res) == yaml.safe_load(expected)
+
+    def test_with_nested_detections(self, pipeline, sigma_sdyaml_backend):
+        rule_id = uuid.uuid4()
+        raw_rule = f"""
+        title: Test Title
+        id: {rule_id}
+        logsource:
+            category: process_creation
+            product: windows
+        detection:
+            selection_img:
+                - Image|endswith: '\WMIC.exe'
+                - OriginalFileName: 'wmic.exe'
+            selection_cli:
+                CommandLine|contains: '/node:'
+            filter_localhost:
+                CommandLine|contains:
+                    - '/node:127.0.0.1 '
+                    - '/node:localhost '
+            condition: all of selection_* and not 1 of filter_*
+        """
+
+        expected = f"""
+        AnalysisType: rule
+        Description: null
+        Detection:
+        - All:

```

Comment:
Based on the thread in Slack, this is just aesthetic and not functional right?

Reply:
yep